pyproject.toml

```diff
@@ -68,7 +68,9 @@ dependencies = [
     "ephem",  # Required for overpass predictor
     "isodate",  # Required for overpass predictor
     "satpy",  # efficiency improvements >= 0.33.1
-    "numexpr",  # for efficiency improvements
+    "numexpr<2.8.5",  # for efficiency improvements
```

Comment:
@jsolbrig I think this is ok without testing / etc?  Do we have an associated issue to update once numexpr has their bug fix in?

Reply:
No, I'll create one.